app/lib/router/router.dart

```diff
@@ -147,16 +146,11 @@ final GlobalKey<NavigatorState> shellNavKey = GlobalKey<NavigatorState>(
   debugLabel: 'shell',
 );
 
-final GlobalKey<NavigatorState> spaceNavKey = GlobalKey<NavigatorState>(
-  debugLabel: 'space',
-);
-
 final GlobalKey<NavigatorState> chatShellKey = GlobalKey<NavigatorState>(
   debugLabel: 'chat',
 );
```

Comment:
can't we get rid of this, too? I think that would solve the `isDesktop`-check from before ... but if you want to do that in a follow-up PR/future issue, that is also fine for now.

Reply:
I'm planning to get rid of `chatShellKey` in follow-up PR as there are few `ChatShell` changes require which is managing chat list and chat detail based on the mobile and desktop platform.